megablocks/layers/arguments.py

```diff
@@ -8,6 +8,7 @@
 import torch
 import torch.distributed as dist
 import torch.nn.functional as F
+import triton
```

Comment:
can you move all the triton imports to be inline and try/excepted? triton won't be installed on non GPU. Not 100% sure whether megablocks works at all on non GPU, but good to avoid unnecessary things that cause it not to

Reply:
I moved all the triton imports inline, but historically, it seems that `triton` has been part of `install_requires` instead of `gpu` specific installation so this shouldn't really be an issue right?

Reply:
yup, you're right. well, no harm done.